webapp/package.json

```diff
@@ -27,6 +27,7 @@
     "async": "^2.6.1",
     "bikram-sambat-bootstrap": "^1.4.2",
     "bootstrap": "^3.3.7",
+    "bootstrap-datepicker": "1.6.x",
```

Comment:
This can lead to issues if Enketo bump their dependency and we forget to bump ours and I don't think it's necessary. If you have trouble browserifying the require calls you can add aliases to the Grunt config to make it work.